juniper_official/Kernel/check-client-limit-reached.rule

```diff
@@ -0,0 +1,49 @@
+/*
+ * This rule checks if any application has opened more than the allowed number
+ * of Ifstate based routing sockets.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-client-limit-reached {
+            sensor kernel-ifstate-sensor {
+                open-config {
+                    sensor-name /junos/kernel-ifstate;
+                    frequency 4m;
+                }
+            }
+            field client-limit-reached {
+                sensor kernel-ifstate-sensor {
+                    path /junos/kernel-ifstate/client-limit-reached;
+                }
+                type integer;
+                description "Client limit reached";
+                }
+            trigger report-ifstate-client-limit-reached {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed